src/c++/vdo/fake/linux/wait.h

```diff
@@ -14,15 +14,16 @@
 
 #include "thread-utils.h"
 
+/* Fields which are not used in VDO unit tests are excluded. */
 typedef struct wait_queue_head {
 	spinlock_t		lock;
 	struct list_head	head;
 } wait_queue_head_t;
 
-struct wait_queue_entry {
-	void		       *private;
+typedef struct wait_queue_entry {
+	void                    *private;
```

Comment:
I think this line is misaligned. the 'p' or "private" should line up with the 'e' of "entry".


Replacing tabs with spaces is good, though.

Reply:
Fixed.